### ISSUE TITLE
Console REPL does not tolerate bugs

### ISSUE THREAD

**Issue by Dakta** [creesch/reddit-moderator-toolbox#483]
Just don't make any typos, K.

**Comment by andytuba** [comment]
Console REPL should ☑its privilege.

**Comment by agentlame** [comment]
The fuck is this about?

**Comment by creesch** [comment]
I dunno, just don't make any typos.

**Comment by agentlame** [comment]
So.... uh.... can we close this?  I don't get it.  What is the issue and what is the proposed solution?

**Comment by Dakta** [comment]
The issue is that the REPL console just breaks on certain errors. It should be more robust. This is the reminder to make it more robust, so we don't have to reload the page when we break it.

**Comment by agentlame** [comment]
Fair enough.  Deferring it, since it's something that only we use for the most part.  And the fact that the native browser console is much more robust.
